tech/languages/ruby/gems-installation.md

```diff
@@ -73,3 +73,32 @@ To look whether the required gem is available as an RPM package, look at
 [IsItFedoraRuby.com](http://isitfedoraruby.com/) site which tracks the Ruby
 integration within the Fedora project by listing available information about
 packaged gems.
+
+## Configuring RubyGems
+
+Fedora configures ruby to install gems on your user's directory; as opposed to the system's, by default. The gems get installed at: `$HOME/.local/share/gem`. 
+
+One can check the defaults [here](https://src.fedoraproject.org/rpms/ruby/blob/rawhide/f/operating_system.rb).
+
```

Comment:
Here is a good place to mention where is the config file or where to create/edit what we are talking about in this section ;)